src/rpc/blockchain.cpp

```diff
@@ -799,7 +799,7 @@ static CBlock GetBlockChecked(const CBlockIndex* pblockindex)
     if (!ReadBlockFromDisk(block, pblockindex, Params().GetConsensus())) {
         // Block not found on disk. This could be because we have the block
         // header in our index but don't have the block (for example if a
```

Comment:
I think we can just remove everything in the parens here. Presumably if a peer sends us headers and the users requests that block over RPC before we've downloaded it we'll hit this condition too?

Reply:
thanks. Shortened the comment a bit. Also force pushed a typo fix and fixed a test failure.